lib/backend.ts

```diff
@@ -0,0 +1,68 @@
+import { connectionDB } from "./initialize_db";
+import { v4 as uuidv4 } from 'uuid';
+
+
+export async function Add_priscription_DB(prescription: string, patientAdress: string) {
```

Comment:
in JS/TS we mostly use `camelCasing` instead of underscores, so `addPrescription`

Reply:
Also, I'm not sure we need this method as the prescription will be added directly to the blockchain. What we need is linking the `prescriptionId` and the `patientAddress`, so maybe replace the type `string` to a type `number`